includes/PageViews.php

```diff
@@ -23,12 +23,20 @@ public function load_scripts() {
 
         echo '<script type="text/javascript">
             jQuery(document).ready( function($) {
-                var data = {
-                    action: "dokan_pageview",
-                    _ajax_nonce: "' . esc_html( $nonce ) . '",
-                    post_id: ' . get_the_ID() . ',
-                }
-                $.post( "' . esc_url( admin_url( 'admin-ajax.php' ) ) . '", data );
+				if(localStorage){
+                    var date = new Date();
+                    var new_date = date.getDate();
+                    var old_date = localStorage.getItem("dokan_product_viewed_' . get_the_ID() . '");
+                    if (new_date != old_date){
+                        var data = {
+                            action: "dokan_pageview",
+                            _ajax_nonce: "' . esc_html( $nonce ) . '",
+                            post_id: ' . get_the_ID() . ',
+                        }
+                        $.post( "' . esc_url( admin_url( 'admin-ajax.php' ) ) . '", data );
+                        localStorage.setItem("dokan_product_viewed_' . get_the_ID() . '", new_date);
+                    }
+				}
```

Comment:
What happens for existing users? which one used COOKIE

Reply:
I'm trying to make an upgrader file asap as you told me.

Reply:
This will not make any big effects on our existing users.